examples/ConfigurableFirmataWiFi/ConfigurableFirmataWiFi.ino

```diff
@@ -339,12 +368,18 @@ DigitalOutputFirmata digitalOutput;
 #include <AnalogInputFirmata.h>
 AnalogInputFirmata analogInput;
 
+// analogWrite not supported for ESP32 boards
+#ifndef ESP32
```

Comment:
I hate to have to do this... will probably not merge this branch until `analogWrite` is added to the esp32 library.